modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/model/DDMFormInstanceSettings.java

```diff
@@ -31,7 +31,7 @@
 	rules = {
 		@DDMFormRule(
 			actions = {
-				"setEnabled('expirationDate', NOT(getValue('neverExpire')))",
+				"setEnabled('expirationDate', equals(getValue('neverExpire'), FALSE))",
```

Comment:
@rodrigopaulino and @diogocordeiro, can we create a test to prevent this issue occur again?

Reply:
@rodrigopaulino, can we talk about it after our daily meeting today? I think we can discuss it with Diogo and perhaps Milton in order to decide the best alternative. :slightly_smiling_face: